### PR TITLE
[Ruijie] Add new platform B6510-48VS8CQ

### PR DESCRIPTION
**Pull request by tim-rj**
### Description of PR

Due to https://github.com/Azure/sonic-buildimage/pull/4954, now add Ruijie hwsku B6510-48VS8CQ into sonic-mgmt.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Add new Ruijie platform
#### How did you verify/test it?
Run tests against B6510-48VS8CQ
#### Any platform specific information?
This change is Ruijie platform specific

### CODE REVIEW

**Comment by stephengzh** [comment]
@wangxin @yxieca  Please review!

File: ansible/module_utils/port_utils.py [hunk 1]
```diff
@@ -198,6 +198,11 @@ def get_port_alias_to_name_map(hwsku, asic_id=None):
         else:
             for i in range(1,9):
                 port_alias_to_name_map["Ethernet1/%d" % i] = "Ethernet%d" % ((i - 1) * 4)
+    elif hwsku == "B6510-48VS8CQ":
+        for i in range(1,48):
```
Comment (neethajohn) [inline ansible/module_utils/port_utils.py:202]: range(1,49) to include 48 as well

File: ansible/module_utils/port_utils.py [hunk 2]
```diff
@@ -198,6 +198,11 @@ def get_port_alias_to_name_map(hwsku, asic_id=None):
...
+            port_alias_to_name_map["twentyfiveGigE0/%d" % i] = "Ethernet%d" % i
+        for i in range(49,56):
```
Comment (neethajohn) [inline ansible/module_utils/port_utils.py:204]: range(49,56) to include 56 as well

File: ansible/module_utils/port_utils.py (hunk 1 above)
Comment (tim-rj) [reply ansible/module_utils/port_utils.py:202]: Our device Ethernet1~Ethernet48 are 25G, Ethernet49~Ethernet56 are 100G

Comment (tim-rj) [reply ansible/module_utils/port_utils.py:202]: range(1,48), not range(1,49)

File: ansible/module_utils/port_utils.py (hunk 2 above)
Comment (tim-rj) [reply ansible/module_utils/port_utils.py:204]: The device port from Ethernet49 to Ethernet 56 are 100G.

File: ansible/module_utils/port_utils.py (hunk 1 above)
Comment (neethajohn) [reply ansible/module_utils/port_utils.py:202]: Probably I didn't specify my comment clearly. port 48 also need to get included in your first range. if you specify range(1,48), only ports 1 to 47 are included. python range does not include the end value

File: ansible/module_utils/port_utils.py (hunk 2 above)
Comment (neethajohn) [reply ansible/module_utils/port_utils.py:204]: The same comment as above. should be range(49,57)

File: ansible/module_utils/port_utils.py (hunk 1 above)
Comment (tim-rj) [reply ansible/module_utils/port_utils.py:202]: Oh! My mistake, I got it.

File: ansible/module_utils/port_utils.py (hunk 2 above)
Comment (tim-rj) [reply ansible/module_utils/port_utils.py]: Fixed , retesting.

File: ansible/module_utils/port_utils.py (hunk 1 above)
Comment (tim-rj) [reply ansible/module_utils/port_utils.py]: Retesting vs now

**Review by neethajohn — APPROVED** [review]

**Pull request merged**